lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -45,6 +45,23 @@ Function Get-DomainMembershipMatch {
 
         return $domain_match
     }
+    catch [System.Security.Authentication.AuthenticationException] {
+        Write-DebugLog "Checking to see if ansible_user account is local and attempt a different method."
```

Comment:
Should mention that it failed to get the computer domain and trying a different method